Makefile

```diff
@@ -18,10 +22,10 @@ check-license:
 	@echo "checking license headers"
 	@./.tool/check-license
 
-tools:
-	go build ./cmd/oci-create-runtime-bundle
-	go build ./cmd/oci-unpack
-	go build ./cmd/oci-image-validate
+tools: $(TOOLS)
+
+$(TOOLS): oci-%:
```

Comment:
Here the oci-% is looking for dependencies of the target that have the oci-% pattern. Is that intended?